storage/client/client.go

```diff
@@ -82,6 +82,8 @@ func (c *StorageClient) Status(ctx context.Context) (*Status, error) {
 		)
 		return nil, common.ErrStorageError
 	}
+	s.LatestUpdate = s.LatestUpdate.UTC()
```

Comment:
Does this clip the precision to one second? The docs don't mention it, the name doesn't imply it.

Edit: Ah I misread the PR description. Yes, +1 to having the same precision in both endpoints. I'm actually surprised that the node truncates the sub-second part in `oasis-node control status`, but I guess we best follow their lead. So let's truncate here, and comment why.

Reply:
> I'm actually surprised that the node truncates the sub-second part in oasis-node control status, but I guess we best follow their lead.

Nice. I didn't even check that.

Reply:
Ugh, there is additional goofiness. We had mentioned this was an [RFC 3339](https://github.com/tendermint/tendermint/issues/6233) timestamp. https://github.com/oasisprotocol/oasis-indexer/blob/e17cca44637fe654934cd4d482fb780b9ba3d2f2/api/spec/v1.yaml#L568

However, we aren't super consistent. I'm gonna stay with UTC and amend the spec. I think staying with `oasis-node control status` is best which I believe is on `UTC`.